lldb/source/Plugins/SymbolFile/NativePDB/SymbolFileNativePDB.cpp

```diff
@@ -501,7 +501,13 @@ lldb::FunctionSP SymbolFileNativePDB::CreateFunction(PdbCompilandSymId func_id,
     return nullptr;
 
   PdbTypeSymId sig_id(proc.FunctionType, false);
-  Mangled mangled(proc.Name);
+
+  std::optional<llvm::StringRef> mangled_opt =
+      FindMangledSymbol(SegmentOffset(proc.Segment, proc.CodeOffset));
+  if (mangled_opt)
+    mangled_opt = StripMangledFunctionName(*mangled_opt, proc.FunctionType);
```

Comment:
Can we call `StripMangledFunctionName` from `FindMangledSymbol`? So we don't need to do it in two places? (then you don't need to call it from `StripMangledFunctionName` either)